docs/DeveloperGuide.md

```diff
@@ -720,101 +765,88 @@ testers are expected to do more *exploratory* testing.
 
    1. Other incorrect delete commands to try: `delete`, `delete x`, `...` (where x is larger than the list size)<br>
       Expected: Similar to previous.
-
-[//]: # (1. _{ more test cases …​ }_)
-
-### Saving data
-
-1. Dealing with missing/corrupted data files
-
-   1. _{explain how to simulate a missing/corrupted file, and the expected behavior}_
-
-[//]: # (1. _{ more test cases …​ }_)
-
+   
 ### Adding a guest
 
-1. Adding a guest
-
-   1. Prerequisites: Only one guest to be added.
-      The name of the guest to be added should not exist in GuestBook.
+Prerequisites: Only one guest to be added.
+      The name and room number of the guest to be added should not exist in GuestBook.
    The format and data of the command should be valid.
 
-   2. Test case: `add n/John Doe p/98765432 e/johnd@example.com rm/05-73
+   1. Test case: `add n/John Doe p/98765432 e/johnd@example.com rm/05-73
                   dr/13/09/22 - 15/09/22 ng/1 rq/Apply for room service `<br>
-      Expected: Guest add successfully
+      Expected: Guest add successfully.
 
    3. Test case: `add n/John Doe p/98765431 e/johnd@nus.com rm/06-73
                   dr/13/09/22 - 15/09/23 ng/1 rq/Kill the insect `<br>
       Expected: No guest is added,
-      because the name is already in GuestBook. Error details shown in the status message.
+      because the **name is already in GuestBook**. Error details shown in the status message.
```

Comment:
Should we use code blocks for all the mentions of fields here? like `Name`, `Phone`, `Email`, `Room`, `Date Range` and `Number Of Guests`?

Reply:
Agreed.